fastlane_core/lib/fastlane_core/itunes_transporter.rb

```diff
@@ -273,7 +273,7 @@ def self.hide_transporter_output?
     def initialize(user = nil, password = nil, use_shell_script = false)
       # Xcode 6.x doesn't have the same iTMSTransporter Java setup as later Xcode versions, so
       # we can't default to using the better direct Java invocation strategy for those versions.
-      use_shell_script ||= Helper.xcode_version.start_with?('6.')
+      use_shell_script ||= Helper.is_mac? and Helper.xcode_version.start_with?('6.')
```

Comment:
The order of the operators is wrong here, please use `&&` instead, so that the tests succeed again 👍